<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registerr</class>
 <widget class="QWidget" name="Registerr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>650</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #313338;
</string>
   </property>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GoidaZvonLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'gg sans semibold';
font-size: 30px;
line-height: 36px;

color: #FFFFFF;

</string>
    </property>
    <property name="text">
     <string>GoidaZvon</string>
    </property>
   </widget>
   <widget class="QWidget" name="RegisterWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>126</y>
      <width>400</width>
      <height>390</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #707070;
border-radius: 15px;
</string>
    </property>
    <widget class="QLabel" name="RegisterLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>380</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans semibold';
font-size: 20px;
line-height: 24px;

color: #FFFFFF;

</string>
     </property>
     <property name="text">
      <string>Register your account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="EmailLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans semibold';
font-size: 16px;
line-height: 12px;

color: #FFFFFF;

</string>
     </property>
     <property name="text">
      <string>Email address</string>
     </property>
    </widget>
    <widget class="QLabel" name="UsernameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans semibold';
font-size: 16px;
line-height: 12px;

color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans semibold';
font-size: 16px;
line-height: 12px;

color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="RepeatPasswordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans semibold';
font-size: 16px;
line-height: 12px;

color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Repeat password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmailInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #373333;
border: 0.5px solid #FFFFFF;
border-radius: 7px;

font-family: 'gg sans semibold';
font-size: 16px;
color: #FFFFFF;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #373333;
border: 0.5px solid #FFFFFF;
border-radius: 7px;

font-family: 'gg sans semibold';
font-size: 16px;
color: #FFFFFF;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #373333;
border: 0.5px solid #FFFFFF;
border-radius: 7px;

font-family: 'gg sans semibold';
font-size: 16px;
color: #FFFFFF;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RepeatPasswordInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #373333;
border: 0.5px solid #FFFFFF;
border-radius: 7px;

font-family: 'gg sans semibold';
font-size: 16px;
color: #FFFFFF;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #5D5656;
border-radius: 10px;

font-family: 'gg sans semibold';
font-size: 14px;
line-height: 17px;

color: #FFFFFF;

</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(72, 127, 198, 1);
border-radius: 10px;

font-family: 'gg sans semibold';
font-size: 18px;
line-height: 19px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordCheckLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>210</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'gg sans normal';
font-size: 14px;
line-height: 15px;

color: #FFFFFF;

</string>
     </property>
     <property name="text">
      <string>Passwords aren't the same</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
